assets/js/page/instellingen.ts

```diff
@@ -9,13 +10,163 @@ const instellingVeranderd = () => {
 		.forEach((el) => el.classList.remove('d-none'));
 };
 
+const urlBase64ToUint8Array = (base64String: string) => {
+	const padding = '='.repeat((4 - (base64String.length % 4)) % 4);
+	const base64 = (base64String + padding)
+		.replace(/\-/g, '+')
+		.replace(/_/g, '/');
+
+	const rawData = window.atob(base64);
+	const outputArray = new Uint8Array(rawData.length);
+
+	for (let i = 0; i < rawData.length; ++i) {
+		outputArray[i] = rawData.charCodeAt(i);
+	}
+	return outputArray;
+};
+
+const applicationServerKey =
+	'BK6nL-UD-kjzpFWXJ6NFkiPEzUEH4diS2BkXBr4ctRz2NU4nyUWZzxLTF2Dulf5spE4EEYVMY2jNmkXhUBTFz2k';
```

Comment:
Deze key hoort niet hier denk ik ;)

Reply:
Het is wel een public key maar het is wel netter in de env ja haha.

Reply:
Je wil m natuurlijk wel makkelijk kunnen veranderen, zonder een nieuwe versie hoeven te pushen. En de private kant van de keypair zou alleen op syrinx moeten staan.